1.0.0-alpha.5 Missing Dependencies

Testing out 1.0.0-alpha.5...

**Current package version**
- Bull => 3.13.0
- BullMQ => Not installed

**Steps Taken**

1.`npm i bull-board@1.0.0-alpha.5`
2. `tsc -p project_src`

However tsc fails and I get the error below

`node_modules/bull-board/dist/index.d.ts:2:34 - error TS2307: Cannot find module 'bullmq'.
2 import { Queue as QueueMq } from 'bullmq';`

`node_modules/bull-board/dist/index.d.ts:3:30 - error TS2307: Cannot find module 'express-serve-static-core'.`
`3 declare const router: import("express-serve-static-core").Express;`

**Note** I tested _1.0.0-alpha.4_ to confirm and got the same error

Also manually installed _bullmq_ package and error did go away however I received this new error

`node_modules/bullmq/dist/interfaces/sandboxed-job.d.ts:3:57 - error TS2304: Cannot find name 'Omit'.`

`3 export interface SandboxedJob<T = any, R = any> extends Omit<JobJson, 'data' | 'opts' | 'progress' | 'log' | 'returnValue'> {`

--- a/package.json
+++ b/package.json
@@ -39,6 +39,9 @@
     "test": "yarn ts-node src/dockest"
   },
   "dependencies": {
+    "@types/bull": "^3.12.1",
+    "@types/express": "^4.17.3",
+    "@types/react": "^16.9.23",
     "date-fns": "2.10.0",
     "ejs": "3.0.1",
     "express": "4.17.1",
@@ -49,12 +52,9 @@
     "redis-info": "^3.0.7"
   },
   "devDependencies": {
-    "@types/bull": "^3.12.0",
-    "@types/express": "^4.17.2",
     "@types/jest": "^24.9.0",
     "@types/node": "^13.1.8",
     "@types/pretty-bytes": "^5.2.0",
-    "@types/react": "^16.9.18",
     "@types/react-dom": "^16.9.5",
     "@types/react-highlight": "^0.12.2",
     "@types/redis-info": "^3.0.0",

--- a/package.json
+++ b/package.json
@@ -41,7 +41,7 @@
   "dependencies": {
     "@types/bull": "^3.12.1",
     "@types/express": "^4.17.3",
-    "@types/react": "^16.9.23",
+    "@types/react-dom": "^16.9.5",
     "date-fns": "2.10.0",
     "ejs": "3.0.1",
     "express": "4.17.1",
@@ -55,7 +55,6 @@
     "@types/jest": "^24.9.0",
     "@types/node": "^13.1.8",
     "@types/pretty-bytes": "^5.2.0",
-    "@types/react-dom": "^16.9.5",
     "@types/react-highlight": "^0.12.2",
     "@types/redis-info": "^3.0.0",
     "@types/supertest": "^2.0.8",

